pkg/rtc/room.go

```diff
@@ -372,13 +372,14 @@ func (r *Room) ResumeParticipant(p types.LocalParticipant, responseSink routing.
 	p.CloseSignalConnection()
 	p.SetResponseSink(responseSink)
 
-	if err := p.SendReconnectResponse(&livekit.ReconnectResponse{
-		IceServers:          iceServers,
-		ClientConfiguration: p.GetClientConfiguration(),
-	}); err != nil {
+	reconnectResponse := r.createReconnectResponseLocked(p, iceServers)
+
+	if err := p.SendReconnectResponse(reconnectResponse); err != nil {
+		prometheus.ServiceOperationCounter.WithLabelValues("participant_resume", "error", "send_response").Add(1)
 		return err
 	}
 
+	// unclear if this is still needed when the participants are being sent down in the reconnect response
```

Comment:
Need to have this in here till all clients update. Maybe worth doing a protocol bump and checking for ability to handle full version of reconnect response and do this only if client cannot handle it.

Reply:
protocol bump and conditional handling sounds good to me!